dm/tests/many_tables/run.sh

```diff
@@ -25,12 +27,24 @@ function incremental_data() {
 			run_sql "INSERT INTO many_tables_db.t$i VALUES ($j,${j}000$j),($j,${j}001$j);" $MYSQL_PORT1 $MYSQL_PASSWORD1
 		done
 	done
+
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"pause-task test" \
+		"\"result\": true" 2
+
+	run_sql "ALTER TABLE many_tables_db.t1 ADD x datetime DEFAULT current_timestamp;" $MYSQL_PORT1 $MYSQL_PASSWORD1
```

Comment:
not sure whether up/down stream uses different tz, if not, can refer this https://github.com/pingcap/tiflow/blob/9f92cb6ca9298b5b6c212e89754edb79b180847d/dm/tests/start_task/run.sh#L71-L77